crates/resolver/src/error.rs

```diff
@@ -166,9 +166,9 @@ impl ResolveError {
             | response_code @ ResponseCode::BADALG
             | response_code @ ResponseCode::BADTRUNC
             | response_code @ ResponseCode::BADCOOKIE => {
-                let mut response = response;
+                let response = response;
```

Comment:
Both this statement and the one below seem pretty pointless, can we just remove them?

Reply:
Done.